docsk8s/administering/ls-k8s-stack-monitoring-external.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[ls-k8s-monitor-external]]
+==== Ship metrics to external {es} cluster
+
+NOTE: The prerequisite of the example is having ECK installed. Checkout <<qs-set-up>>
```

Comment:
Is it a prerequisite to have ECK installed? Or just the CRDs?

Reply:
I think I get confused between `Metrics can be sent to an {es} cluster that is not managed by ECK`, and the prerequisite of ECK for this - one use case is for a new logstash cluster sending to a pre-existing monitoring cluster, not migrated to ECK

Reply:
You are right, to config the external ES we just need CRDs. I mess up with another example.